src/customParser.ts

```diff
@@ -0,0 +1,330 @@
+import BoxGroup from './geometry/BoxGroup';
+import CircleGroup from './geometry/CircleGroup';
+import ConeGroup from './geometry/ConeGroup';
+import * as THREE from 'three';
+
+const lookup = [1, 2];
+for (let i = 2; i <= 77; i++) {
```

Comment:
dafuq?

Reply:
Renamed it to fibonacciLookup (it's a lookup table)